HappyAnding/HappyAnding/Views/TabView/ShortcutTabView.swift

```diff
@@ -64,6 +67,11 @@ struct ShortcutTabView: View {
                             }
                             self.tappedTwice = false
                         })
+                        .onChange(of: isFolded, perform: { tappedMore in
+                            withAnimation {
+                                proxy.scrollTo(999, anchor: .bottom)
+                            }
+                        })
```

Comment:
후행 클로저를 사용하면 더 깔끔할 것 같아요!
```suggestion
                        .onChange(of: isFolded) { tappedMore in
                            withAnimation {
                                proxy.scrollTo(999, anchor: .bottom)
                            }
                        }
```